packages/dd-trace/test/config.spec.js

```diff
@@ -243,8 +244,8 @@ describe('Config', () => {
       { name: 'site', value: 'datadoghq.com', origin: 'default' },
       { name: 'hostname', value: '127.0.0.1', origin: 'default' },
       { name: 'port', value: '8126', origin: 'default' },
-      { name: 'debug', value: false, origin: 'default' },
```

Comment:
does debug not exists anymore ?

Reply:
It is not present in the first argument of the call to `updateConfig`.